src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceComplaintsAsserts.java

```diff
@@ -117,4 +118,13 @@ public void verifyCriteriaWasAssociatedToComplaint(final String criteria) {
         String actualValue = SurveillanceComplaintsPage.associatedCriteriaValue(getDriver()).getText();
         assertTrue(actualValue.contains(criteria), "Expect " + criteria + " to be found in " + actualValue);
     }
+
+    @Then("^I see \"([^\"]*)\" as the page title on Complaints Reporitng page$")
+    public void iSeeComplaintsReportingPageTitle(final String expectedPageTitle) {
+        WebElement link = SurveillanceComplaintsPage.complaintsReportingPageTitle(getDriver(), expectedPageTitle);
```

Comment:
why not use getTitle() and assert?

Reply:
I'm using getText() instead of getTitle() and it's doing the same thing, please let me know if you want me to use getTitle() instead of getText()

Reply:
It just seemed simple to use getTitle() instead of defining page object, getting text, storing it in a string and compare. Also, i am not entirely sure why WebElement link def is needed here. Does getText() of complaintsReportingPageTitle and asset (last two lines in this method) not work? The text is at top of the page, so I was thinking it should be visible when page loads.

Reply:
Used getTitle() and verified locally - works as expected